css/style.css

```diff
@@ -0,0 +1,281 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;  
+}
+
+@font-face {
+    font-family: 'myFont';
+    src: url(myFont.otf);
+}
```

Comment:
so I can delete all of this: 
@font-face {
    font-family: 'myFont';
    src: url(myFont.otf);
}

because I have this on the html: (<link href="https://fonts.cdnfonts.com/css/blanka" rel="stylesheet"> ) ?